deploy/examples/operator-openshift.yaml

```diff
@@ -753,4 +754,11 @@ spec:
           emptyDir: {}
         - name: webhook-cert
           emptyDir: {}
+      securityContext:
```

Comment:
The CI doesn't run the tests in an environment like Openshift where the security context would be strictly enforced. We'll need to confirm any changes to the security context manually on Openshift. What tests have you run there?

Reply:
That's tough. It is possible to enforce the security policy for a namespace with a label. Maybe an action item should be to make a test with this label set?

https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces

Reply:
Fundamentally, what is the goal of this change? As you noted in #11755, the namespace policy will still be required the run privileged. The ceph daemons have the requirement to run privileged, there is no way around that. 
The operator doesn't need to run privileged so we can certainly proceed with this PR, but it will be important to test thoroughly to make sure it's not broken in different environments. This is always the challenge with security changes.

Reply:
I don't disagree with the desire to harden, it will just take some dedication to verify the different environments.